Ports/abseil/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=abseil
+useconfigure=true
+version=20230802.0
+depends=(  )
```

Comment:
Wat?

Reply:
Didn't delete that at first to see what abseil might depend on.